agents/agent.go

```diff
@@ -44,13 +45,39 @@ Only return the JSON object and nothing else.
 }
 
 func LoadPersonas() ([]Agent, error) {
-	personaJSON, err := os.ReadFile("../agents/persona.json")
-	if err != nil {
-		return nil, fmt.Errorf("failed to read persona.json: %w", err)
+	// Try multiple paths to find persona.json
+	possiblePaths := []string{
+		"agents/persona.json",    // From project root
```

Comment:
Can we move persona.json to a common path that could be recognized both in the development environment as well as in the standalone binary?